pkgs/servers/dns/bind/default.nix

```diff
@@ -33,6 +32,8 @@ stdenv.mkDerivation rec {
     "--without-python"
   ];
 
+  postInstall = "$out/sbin/rndc-confgen -r /dev/urandom -b 256 | head -n 5 > $out/etc/rndc.key";
```

Comment:
That key doesn't belong into the generated derivation. Note that the output from this build will go to `cache.nixos.org` and will be shared between almost all installations.